sublime/README.md

```diff
@@ -26,7 +26,7 @@ If you don't have [Package Control](https://sublime.wbond.net/), install it.
 Install the list of ST3 packages above using Package Control. Restart Sublime.
 
 Open `st3/SublimeLinter.sublime-settings` in this folder and copy it into your SublimeLinter user settings located in:
-* OS X: `~/Library/Application Support/Sublime Text 3/SublimeLinter.sublime-settings`.
+* OS X: `~/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/SublimeLinter.sublime-settings`.
```

Comment:
good catch, but it should be in `~/Library/Application Support/Sublime Text 3/Packages/User/SublimeLinter.sublime-settings`, otherwise it'll get erased when the package updates